src-tool/Pact/Analyze/Check.hs

```diff
@@ -1332,7 +1335,7 @@ verifyCheck de moduleData funName check checkType = do
       case toplevel of
         Left checkFailure -> throwError $ ModuleCheckFailure checkFailure
         Right (TopFun fun _) -> ExceptT $ fmap Right $
-          verifyFunctionProperty checkEnv (mkFunInfo fun) funName checkType $
+          (fmap.fmap.fmap.fmap) fst (verifyFunctionProperty checkEnv (mkFunInfo fun) funName checkType) $
```

Comment:
This is really quite unfortunate. Can we add a comment showing some kind of example expected value, so that the reader can understand what `fst` will apply to?

Reply:
I added a comment. Note, this function `verifyCheck` is only used inside the `AnalyseSpec`. At some point we should clean up this.